### DISCUSSION TITLE
Is it possible to upload videos with the cloudinary upload widget integration?

### DISCUSSION THREAD

**Discussion by qb-rivkyb** [decaporg/decap-cms#6103] [Q&A]
Hi,
I was wondering if anyone has experience with uploading videos to cloudinary using `netlify-cms-app` cloudinary widget integration. I found this article but I don't know if it's still relevant and if it applies to my use case https://cloudinary.com/blog/managing_visual_media_with_the_cloudinary_netlify_integration

Any help appreciated!

**Comment by erezrokah** [comment]
Hi @qb-rivkyb, when you use the Cloudinary integration the CMS passes full control to the Cloudinary JS widget, so whatever the widget can do you can do with the CMS.

I recommend asking Cloudinary support if their standalone widget supports videos.
